src/cluster.ml

```diff
@@ -64,13 +71,23 @@ type service = [
   | `OCamlorg_v3b of string                  (* OCaml website @ v3b.ocaml.org aka www.ocaml.org *)
   | `OCamlorg_v3c of string                  (* Staging OCaml website @ staging.ocaml.org *)
   | `Aws_ecs of Aws.t                        (* Amazon Web Services - Elastic Container Service *)
-]
+] [@@deriving show]
 
 type deploy_info = {
   hub_id : Cluster_api.Docker.Image_id.t;
   services : service list;
 }
 
+let show_service (org, name, builds) =
+  let builds =
+    List.map
+      (fun (build, _deploys) ->
+        Printf.sprintf "%s" (Cluster_api.Docker.Image_id.to_string build))
+      builds
+    |> String.concat "\n"
+  in
+  Printf.sprintf "- %s/%s\n%s" (Build.account org) name builds
+
```

Comment:
I think we can promote the comments on lines 55 to 73 (giving the FQDN) into code, in the form of generation of URLs here (perhaps with an exception for AWS?). Then that could be used in the doc generation to make links. WDYT?

Reply:
This will be included in another PR.